dandischema/datacite.py

```diff
@@ -85,7 +90,7 @@ def to_datacite(meta: ty.Union[dict, PublishedDandiset]) -> dict:
         "resourceTypeGeneral": "Dataset",
     }
     # meta has also attribute url, but it often empty
-    attributes["url"] = meta.url
+    attributes["url"] = str(meta.url)
```

Comment:
could it be `None`? then if to make robust may be:
```suggestion
    attributes["url"] = str(meta.url or "")
```

Reply:
it's a required field, therefore cannot be `None`

Reply:
Ok, then just remember and tell everyone that if we relax that some time (allowing empty but not allowing None is fragile) - we need to fix code here ;-)